app/views/all_casa_admins/dashboard/show.html.erb

```diff
@@ -1,13 +1,23 @@
-<div class="row">
-  <div class="col-sm-12 dashboard-table-header">
-    <h1>All CASA Admin</h1>
-    <a class="btn btn-primary" href="<%= new_all_casa_admins_casa_org_path %>">New CASA Organization</a>
-    <a class="btn btn-primary" href="<%= new_all_casa_admin_path %>">New All CASA Admin</a>
+<div class="title-wrapper pt-30">
+  <div class="row align-items-center">
+    <div class="col-md-6">
+      <div class="title mb-30">
+        <h2>All CASA Admin</h2>
+      </div>
+    </div>
+    <div class="col-md-6">
+      <div class="breadcrumb-wrapper mb-30">
+        <span class="ml-5">
+          <a class="btn-sm main-btn primary-btn btn-hover" href="<%= new_all_casa_admins_casa_org_path %>">New CASA Organization</a>
```

Comment:
I think these should be `link_to` rather than <a>. Also looks like you are missing the + icon the other pages use when adding something.

Reply:
Done